.github/workflows/manual-mainnet-deploy-staker.yml

```
@@ -0,0 +1,37 @@
+      run: |
+        curl -L https://foundry.paradigm.xyz | bash
+        foundryup
+
+    - name: Deploy contract
+      env:
+        RPC_URL: ${{ secrets.MAINNET_RPC_URL }}
+        PRIVATE_KEY: ${{ secrets.PRIVATE_KEY }}
+        STAKE_TOKEN_ADDRESS: ${{ github.event.inputs.stakeTokenAddress }}
+        PAYMENT_TOKEN_ADDRESS: ${{ github.event.inpouts.paymentTokenAddress }}
+      run: |
+        forge script script/Deploy.s.sol:Deploy --rpc-url $RPC_URL --private-key $PRIVATE_KEY 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)